runtime/jvmti/jvmtiHook.c

```diff
@@ -505,6 +509,56 @@ jvmtiHookVirtualThreadUnmount(J9HookInterface **hook, UDATA eventNum, void *even
 
 #endif /* JAVA_SPEC_VERSION >= 19 */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static void
+jvmtiHookVMCheckpoint(J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData)
+{
+	J9CheckpointEvent * data = eventData;
```

Comment:
The same for `data` and `j9env` as https://github.com/eclipse-openj9/openj9/pull/17252/files#r1176818684.